rust/notes.md

```diff
@@ -0,0 +1,54 @@
+# Implementors' notes
+
+## Desired subjects
+
+This is a more full list of topics to cover than
+in the README.
+
+- error handling
+  - simple vs. complex, Fail vs StdError, etc
+- logging w/ log and slog
+  - how env_logger works?
+- trees vs maps
+- async vs sync networking
+- sync file io and solutions to blocking
+- buffered vs unbuffered i/o
+- benchmarking, criterion and critcmp, black_box
+- RUST_BACKTRACE
+- how to find crates
+- where to ask questions
+- futures
+- tokio
+- async/await? - maybe future iterations
+- generic placeholder idiom - let foo: Vec<_> =
+- construction w/ iterators idiom
+- semver trick
+- impl trait, the Into<Option<_>> trick
+- Rust history, culture, design principles
+- rustfmt, clippy, configuring both
+- rust 2018?
+- tools most rust programmers should know
+- build scripts
+- using RUSTFLAGS
+- debugging
+- profiling
+- how not to write Rust (bad practices from other languages)
+- variable shadowing
+- DSTs
+
+## Sources
+
+- https://pdos.csail.mit.edu/6.824/schedule.html
+  - The MIT distributed systems course this course
+    is inspired by and intended to precede
+- https://github.com/brson/rust-three-days-course
```

Comment:
Maybe add @skade 's original course link?

Reply:
It's currently under https://github.com/ferrous-sytems/rust-three-days-course.

Reply:
Oops. I'll change this.